src/GraphQL.Relay.StarWars/GraphQL.Relay.StarWars.csproj

```diff
@@ -6,10 +6,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="GraphQL.MicrosoftDI" Version="5.2.0" />
-    <PackageReference Include="GraphQL.SystemTextJson" Version="5.2.0" />
-    <PackageReference Include="GraphQL.Server.Transports.AspNetCore" Version="6.0.0" />
-    <PackageReference Include="GraphQL.Server.Ui.GraphiQL" Version="6.0.0" />
+    <PackageReference Include="GraphQL.MicrosoftDI" Version="5.3.1" />
+    <PackageReference Include="GraphQL.SystemTextJson" Version="5.3.1" />
```

Comment:
This should be GraphQLTestVersion, right?

Reply:
I don’t know if this gets tested actually. For a sample project, bumping to latest is probably correct

Reply:
> This should be GraphQLTestVersion, right?

It's sample. CI ovewrites graphql version for tests.